src/swap/components/SwapProvider.tsx

```diff
@@ -272,6 +278,7 @@ export function SwapProvider({
       }
       await processSwapTransaction({
         config,
+        maxSlippage,
```

Comment:
In case was miss, let's cover this comment

https://github.com/coinbase/onchainkit/pull/1210/files#r1744336194

Reply:
This link is broken, it is sending me to the top of the updated files page.

Reply:
Also, I don't think is clean to just pass the maxSlippage. Like think 7 steps ahead, statusData will contain other info that need to persist inside processSwapTransaction when setLifeCycleStatus is called.

So we should pass lifeCycleStatus, inside processSwapTransaction.

So that we can persit the info we need from statusData when we recall setLifeCycleStatus.

Reply:
should useAggregator be handled the same way?

Reply:
@Zizzamia Updated to pass in `lifeCycleStatus` instead of `maxSlippage`.

And now within `processSwapTransaction` we are grabbing `maxSlippage`:

```
  const maxSlippage =
    lifeCycleStatus.statusName !== 'error'
      ? lifeCycleStatus.statusData.maxSlippage
      : 3;
```